### PR TITLE
feat: accept any version greater or equal to 1.14.0

### PR DESCRIPTION
**Pull request by Cactusbone**
This accepts maplibre-gl v2 as a peer dependency.

 - [x] briefly describe the changes in this PR
 - [ ] write tests for all new functionality
 - [x] run `npm run docs` and commit changes to API.md
 - [x] update CHANGELOG.md with changes under `master` heading before merging

### CODE REVIEW

**Comment by wipfli** [comment]
Thanks for this. I think it would be good to have the test running in this repo on pull requests. Any chance to review #47 ?

**Comment by wipfli** [comment]
When you update `package.json`, please make sure to also run `npm install` such that the changes are reflected in `package-lock.json`. 

At the moment, I get the following diff: 
```diff
diff --git a/package-lock.json b/package-lock.json
index 0faa298..14875db 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -1,12 +1,12 @@
 {
   "name": "@maplibre/maplibre-gl-geocoder",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@maplibre/maplibre-gl-geocoder",
-      "version": "1.3.0",
+      "version": "1.3.1",
       "license": "ISC",
       "dependencies": {
         "lodash.debounce": "^4.0.6",
@@ -38,7 +38,7 @@
         "node": ">=6"
       },
       "peerDependencies": {
-        "maplibre-gl": "^1.14.0"
+        "maplibre-gl": ">=1.14.0"
       }
     },
     "node_modules/@babel/code-frame": {
```

The part about the plugin version `1.3.1` is related to the fact that we forgot to run `npm install` at the last version bump.

**Comment by Cactusbone** [comment]
ha yeah, forgot this file. saw your PR https://github.com/maplibre/maplibre-gl-geocoder/pull/45 too :)

**Comment by Cactusbone** [comment]
I've rebased to enable CI :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Features / Improvements 🚀
 
-- N/A
+- accept any version greater or equal to 1.14.0 [#49](https://github.com/maplibre/maplibre-gl-geocoder/pull/49)
```
Comment (wipfli) [inline CHANGELOG.md]:
```suggestion
- accept any version of MapLibre GL JS greater or equal to 1.14.0 [#49](https://github.com/maplibre/maplibre-gl-geocoder/pull/49)
```

**Review by wipfli — APPROVED** [review]

**Comment by wipfli** [comment]
Just to double-check, you tried this out with MapLibre GL JS v2.1.8?

**Comment by Cactusbone** [comment]
> Just to double-check, you tried this out with MapLibre GL JS v2.1.8?

I tried with 2.1.6 :)

Code is very similar to https://maplibre.org/maplibre-gl-js-docs/example/geocoder/ which seems to use v2.1.7

**Pull request merged**